### ISSUE TITLE
Example beamChain-1.scad Appears to be Broken

### ISSUE THREAD

**Issue by LenStruttmann** [Irev-Dev/Round-Anything#51]
It appears that example beamChain-1.scad is broken.  I get the following error:

Parsing design (AST generation)...
Compiling design (CSG Tree generation)...
[WARNING: Unable to convert translate(undef) parameter to a vec3 or vec2 of numbers in file beamChain-1.scad, line 16](https://github.com/Irev-Dev/Round-Anything/issues/16,C:/Users/red/Documents/OpenSCAD/libraries/Round-Anything-1.0.4/examples/beamChain-1.scad)
[TRACE: called by 'translate' in file beamChain-1.scad, line 16](https://github.com/Irev-Dev/Round-Anything/issues/16,C:/Users/red/Documents/OpenSCAD/libraries/Round-Anything-1.0.4/examples/beamChain-1.scad)
[TRACE: called by 'linear_extrude' in file beamChain-1.scad, line 13](https://github.com/Irev-Dev/Round-Anything/issues/13,C:/Users/red/Documents/OpenSCAD/libraries/Round-Anything-1.0.4/examples/beamChain-1.scad)
Execution aborted

I am running OpenSCAD daily OpenSCAD-2024.03.23-x86-64.

**Comment by JeroenEeuwes** [comment]
I just installed a snap of openscad-nightly (from https://snapcraft.io/install/openscad-nightly/ubuntu ) and it does work, but only if I tell it where the libraries are located. SO from my terminal/console I need to run this:
`OPENSCADPATH=~/.local/share/OpenSCAD/libraries openscad-nightly`

Mine also gives an warning on line 16. That warning goes away if I change line 16 to this:
`  translate([0,0,0]){`
Perhaps in your version this warning is somehow preventing the rendering altogether?

**Comment by LenStruttmann** [comment]
Yes, mine does not render because I have the option **Stop on first warning** enabled.  It was this warning that I was reporting.

**Comment by JeroenEeuwes** [comment]
> Yes, mine does not render because I have the option **Stop on first warning** enabled. It was this warning that I was reporting.

I don't think that's the default setting in OpenSCAD. I think it would be helpful if you mentioned in your first message that you already knew it was due to the warning on the translate keyword and that you setup your OpenSCAD to stop on warnings. http://www.catb.org/~esr/faqs/smart-questions.html

I got that warning in my normal OpenSCAD installation too, so I wouldn't have had to install a nightly build to see if something was changed in OpenSCAD itself.

If you change the line I mentioned above in your system it should work.
